install/installer/pkg/components/image-builder-mk3-wsman/deployment.go

```diff
@@ -157,17 +157,6 @@ func deployment(ctx *common.RenderContext) ([]runtime.Object, error) {
 					RestartPolicy:                 corev1.RestartPolicyAlways,
 					TerminationGracePeriodSeconds: pointer.Int64(30),
 					Volumes:                       volumes,
-					InitContainers: []corev1.Container{
-						{
-							Name:  "setup",
-							Image: ctx.ImageName(ctx.Config.Repository, imagebuildermk3.Component, ctx.VersionManifest.Components.ImageBuilderMk3.Version), ImagePullPolicy: corev1.PullIfNotPresent,
-							Args: []string{
-								"setup",
-							},
-							SecurityContext: &corev1.SecurityContext{RunAsUser: pointer.Int64(0)},
-							Env:             common.ProxyEnv(&ctx.Config),
-						},
-					},
```

Comment:
@Furisto any idea if we can de-duplicate `image-builder-mk3` and `image-builder-mk3-wsman`? Not the first time a change to the former was missed for the wsman copy